services/brig/test/integration/Federation/End2end.hs

```diff
@@ -238,6 +239,29 @@ testAddRemoteUsersToLocalConv brig1 galley1 brig2 = do
     )
     !!! (const 200 === statusCode)
 
+  -- test GET /conversations/:backend1Domain/:cnv
+  testQualifiedGetConversation galley1 "galley1" alice bob convId
+  testQualifiedGetConversation galley2 "galley2" alice bob convId
```

Comment:
Shouldn't the second call to `testQualifiedGetConversation` have `alice` and `bob` swapped when compared to the other call to the same function? I would expect so, because I would expect a list of `OtherMember`s to be dependent on the beholder.

Reply:
Locally I switched `alice` and `bob` in line 244 and the test still passes, which suggests the test needs improvement.

Reply:
yes, good point.